src/Symfony/Component/Messenger/Transport/Doctrine/DoctrineReceivedStamp.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\Doctrine;
+
+use Symfony\Component\Messenger\Stamp\StampInterface;
+
+/**
+ * @author Vincent Touzet <vincent.touzet@gmail.com>
+ */
+class DoctrineReceivedStamp implements StampInterface
+{
+    private $id;
+
+    /**
+     * @param string $id
+     */
+    public function __construct(string $id)
+    {
+        $this->id = $id;
+    }
+
+    /**
+     * @return array
+     */
```

Comment:
Remove the phpdoc it doesn't add anything that the return type doesn't add (and it's wrong in this case anyways)